### DISCUSSION TITLE
Include a "Does Linux ARM" variant

### DISCUSSION THREAD

**Discussion by calejvaldez** [ThatGuySam/doesitarm#1026] [Ideas]
[Asahi Linux](https://asahilinux.org/) is reverse engineering the M1 chip to support Linux distributions, and [it seems like a release for its Fedora Remix is coming very soon](https://www.youtube.com/watch?v=XIGxKyekvBQ&t=20512). I'm personally very excited about this, however, it does mean that the arm64 compatibility issue is coming up again. I think it would be nice to have a version of the website for Linux ARM.

Some notable apps without Linux ARM support:
- Discord (no surprise)
- Todoist
- Zulip
- Spotify

**Comment by ThatGuySam** [comment]
I'd be open to exploring it. 

We'd need to start with a simple list, and we can use the existing issue review process on this repo. 

We can also look into adding some kind of status on apps for Linux Compatibility note on app pages. 

One of the most powerful parts of Does It ARM is that it lets people scan specific apps not listed on the site and anonymously reports the results for everyone on the internet. 

Ideally, I would want the same mechanism for Linux ARM Application Bundles and Binaries so that people aren't limited to what we can manually review.

**Comment by ThatGuySam** [comment]
@calejvaldez Are there any popular Linux App Repositories like Homebrew that contain app bundles and binaries that we could automatically scan and report on?

**Reply by calejvaldez** [reply]
Hey! Thanks for reaching out. It seems like as of now, [Homebrew for ARM Linux is unsupported](https://docs.brew.sh/Homebrew-on-Linux#arm-unsupported). However, I think the next best thing may be the [Snapstore](https://snapstore.io) or [FlatHub](https://flathub.org). Both of these are very similar to Homebrew, but I think the Snapstore is more widely supported.

Please let me know if there's anything else I can do to help. Thanks!

**Reply by ThatGuySam** [reply]
Yeah, if you want, feel free to see if you can collect anything verifying ARM Support on Linux or see if you can get Asahi Linux running on a machine you manually verify apps with screenshots as describe on the [App Request Submission](https://github.com/ThatGuySam/doesitarm/issues/new?assignees=&labels=New+App+Request&projects=&template=app-request-template.yml)

**Reply by calejvaldez** [reply]
Got it. I'll try to do so sometime before the weekend.
